src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/DraftApprenticeshipController.cs

```diff
@@ -305,6 +298,33 @@ public async Task<IActionResult> ViewEditDraftApprenticeship(DraftApprenticeship
             }
         }
 
+        [HttpGet]
+        [Route("{DraftApprenticeshipHashedId}/recognise-prior-learning")]
+        [Authorize(Policy = nameof(PolicyNames.HasContributorOrAbovePermission))]
+        public async Task<IActionResult> RecognisePriorLearning(Models.RecognisePriorLearningRequest request)
+        {
+            var model = await _modelMapper.Map<RecognisePriorLearningViewModel>(request);
+            return View("RecognisePriorLearning", model);
+        }
+
+        [HttpPost]
+        [Route("{DraftApprenticeshipHashedId}/recognise-prior-learning")]
+        [Authorize(Policy = nameof(PolicyNames.HasContributorOrAbovePermission))]
+        public async Task<IActionResult> RecognisePriorLearning(RecognisePriorLearningViewModel request)
+        {
+            var result = await _modelMapper.Map<RecognisePriorLearningResult>(request);
+            
+            if (result.HasStandardOptions)
```

Comment:
Looking at the mapper implementation, Can HasStandardOptions ever be true?

Reply:
Isn't it set [here](https://github.com/SkillsFundingAgency/das-providercommitments/blob/5f1d631de0db8bd49d6031e740ffd92a1ebfc5b0/src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/RecognisePriorLearningViewModelToResultMapper.cs#L35)?

Reply:
It is!